docs/blog/2022-07-20-bun-is-the-future.mdx

```diff
@@ -0,0 +1,239 @@
+
+As for the new SQLite adapter, I guess the normal way to do sqlite in Node is a bit confusing. Most people combine the older `sqlite3` package with the `sqlite` wrapper to add promise support,
+but it's not that bad. Bun's solution looks a bit cleaner, and I won't even bother checking if it's faster.
+
+
+## The problem communicating all this value
+
+I'm concerned it's going to be a little hard to communicate all this value to the community. Bun is like a complete ecosystem replacement, and that's a big shift. I hope that people will get it.
+
+Bun is still very new and currently has no complete documentation. A [very long readme](https://github.com/oven-sh/bun#readme) is the main source of truth
+right now. It's not that hard to throw a docusaurus site together (like the one you're reading right now) and generate typedoc from the TypeScript types which already has thorough inline comments, so I imagine this will be coming soon.
+
+## Other alternatives
+### Deno
+You can probably skip this part if you don't know what Deno is or don't care. Personally, I'm way more excited about Bun than Deno.
+
+[Deno](https://deno.land/), written by the original creator of Node, does claim to resolve some outstanding issues.
+It makes es-modules the default, adds first-party TypeScript support (no need to transpile NPM modules before publishing), and so on. In my opinion
+it introduces other serious shortcomings along the way. The elephant in the room is that Deno has made enough breaking changes to package resolution and syntax that **it isn't
+compatible with the existing NPM ecosystem.** This means that Deno needs to be a completely new ecosystem of libraries. There has been a bit of development with some early library support,
+but I'd say that clout can only get you so much buy-in, especially when so much momentum is involved.  There are a few other things in Deno that strike me as half-baked, like the [lack of a package.json](https://deno.land/manual/examples/manage_dependencies),
+both from a module resolution perspective, and also because without a manifest file, there is nowhere to write extensible metadata about your package. Even GoLang has added `pom.xml` for this reason.
+
+Also, I just want to point at that the sandboxing / permission system in Deno is the right idea but isn't fine-grained enough. It lives at the top level of the entire project, not at the package level, meaning a big app ends up needing all permissions and you're back to square one.
+As a security company, we were disappointed in its potential to protect large, real world apps from supply-chain attacks. Bun isn't claiming to solve this problem either (yet), I just wanted to vent.
+
+I should probably write a whole post about this, but the short answer is:
+
+### Bun has way more potential to take-off
```

Comment:
It's fair if this is your opinion but deno has funding to solve some of the problems you list below :shrug:. Good product != market success.